module-gui/gui/core/FontManager.cpp

```diff
@@ -96,28 +114,61 @@ namespace gui
         }
     }
 
-    std::vector<std::string> FontManager::getFontsList()
+    std::map<Fonts, std::string> FontManager::getFontsList()
     {
+        auto fd = std::fopen(fontMapFile.c_str(), "r");
```

Comment:
File descriptor ought to be closed if not used anymore

Reply:
added fclose  few lines below